uber/site_sections/preregistration.py

```diff
@@ -162,7 +162,12 @@ def form(self, session, message='', edit_id=None, **params):
                 else:
                     target = group if group.badges else attendee
                     track_type = c.EDITED_PREREG if target.id in self.unpaid_preregs else c.UNPAID_PREREG
-                    self.unpaid_preregs[target.id] = to_sessionized(attendee, group)
+                    if attendee.paid == c.NEED_NOT_PAY:
```

Comment:
What needs to happen here? It seems to be working correctly... but I could be wrong!